### PR TITLE
(UPDATED) Added Finnish (Suomi) language

### PR DESCRIPTION
**Pull request by syorito-hatsuki**
Hi. Here Finnish translate for your mod. It`s not 100% correct but I translate all what I know.

### CODE REVIEW

File: src/main/resources/assets/ironchest/lang/fi_FI.lang [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+################
+
+tile.ironchest.chest.iron.name=Rautainen arkku
+tile.ironchest.chest.gold.name=Kultainen arkku
+tile.ironchest.chest.diamond.name=Timanttinen arkku
+tile.ironchest.chest.copper.name=Kupari arkku
+tile.ironchest.chest.silver.name=Hopea arkku
+tile.ironchest.chest.crystal.name=Kristallinen arkku
+tile.ironchest.chest.obsidian.name=Obsidiaani arkku
+tile.ironchest.chest.dirtchest9000.name=Multa arkku 9000!
```
Comment (jikuja) [inline src/main/resources/assets/ironchest/lang/fi_FI.lang:12]: Maybe using one style for all chests is better?

Also compound words: Kupari arkku, Hopea arkku, Obsidiaani arkku and Multa arkku 9000!

File: src/main/resources/assets/ironchest/lang/fi_FI.lang [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+
+item.ironchest.chest.iron_gold.name=Parannus: raudasta kultaisen
+item.ironchest.chest.gold_diamond.name=Parannus: kultaisesta timanttisen
+item.ironchest.chest.copper_silver.name=Parannus: kuparisesta hopean
+item.ironchest.chest.silver_gold.name=Parannus: hopeasta kultaisen
+item.ironchest.chest.copper_iron.name=Parannus: kuparisesta rautaisen
+item.ironchest.chest.diamond_crystal.name=Parannus: timanttisesta krystallisen
+item.ironchest.chest.wood_iron.name=Parannus: puusta rautaisen
+item.ironchest.chest.wood_copper.name=Parannus: puisesta kuparisen
+item.ironchest.chest.diamond_obsidian.name=Parannus: timanttisesta obsidiaanisen
```
Comment (jikuja) [inline src/main/resources/assets/ironchest/lang/fi_FI.lang]: Parannus or päivitys?

File: src/main/resources/assets/ironchest/lang/fi_FI.lang [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+tile.ironchest.chest.iron.name=Rautainen arkku
+tile.ironchest.chest.gold.name=Kultainen arkku
+tile.ironchest.chest.diamond.name=Timanttinen arkku
+tile.ironchest.chest.copper.name=Kupari arkku
+tile.ironchest.chest.silver.name=Hopea arkku
+tile.ironchest.chest.crystal.name=Kristallinen arkku
+tile.ironchest.chest.obsidian.name=Obsidiaani arkku
+tile.ironchest.chest.dirtchest9000.name=Multa arkku 9000!
+
+item.ironchest.chest.iron_gold.name=Parannus: raudasta kultaisen
```
Comment (jikuja) [inline src/main/resources/assets/ironchest/lang/fi_FI.lang]: raudasta kultaiseen or raudasta kultainen?

raudasta kultaisen is kind of funny. Same comment for all.

File: src/main/resources/assets/ironchest/lang/fi_FI.lang [hunk 4]
```diff
@@ -0,0 +1,177 @@
...
+item.ironchest.chest.gold_diamond.name=Parannus: kultaisesta timanttisen
+item.ironchest.chest.copper_silver.name=Parannus: kuparisesta hopean
+item.ironchest.chest.silver_gold.name=Parannus: hopeasta kultaisen
+item.ironchest.chest.copper_iron.name=Parannus: kuparisesta rautaisen
+item.ironchest.chest.diamond_crystal.name=Parannus: timanttisesta krystallisen
```
Comment (jikuja) [inline src/main/resources/assets/ironchest/lang/fi_FI.lang]: Kristalli or krystalli.

File: src/main/resources/assets/ironchest/lang/fi_FI.lang [hunk 5]
```diff
@@ -0,0 +1,177 @@
...
+tile.ironchest.shulker_box.obsidian.purple.name=Obsidiaani Shulker-laatikko
+tile.ironchest.shulker_box.obsidian.blue.name=Sininen Obsidiaani Shulker-laatikko
+tile.ironchest.shulker_box.obsidian.brown.name=Ruskea Obsidiaani Shulker-laatikko
+tile.ironchest.shulker_box.obsidian.green.name=Vihreä Obsidiaani Shulker-laatikko
+tile.ironchest.shulker_box.obsidian.red.name=Punainen Obsidiaani Shulker-laatikko
+tile.ironchest.shulker_box.obsidian.black.name=Musta Obsidiaani Shulker-laatikko
+
+item.ironchest.shulker_box.iron_gold.name=Parannus: Rautaisesta Kultaisen Shulker-laatikko
+item.ironchest.shulker_box.iron_gold.tooltip= 
+item.ironchest.shulker_box.gold_diamond.name=Parannus: золотого в алмазный Шалкеровый ящик
```
Comment (jikuja) [inline src/main/resources/assets/ironchest/lang/fi_FI.lang]: Why russian?

File: src/main/resources/assets/ironchest/lang/fi_FI.lang [hunk 6]
```diff
@@ -0,0 +1,177 @@
...
+item.ironchest.shulker_box.vanilla_copper.name=Parannus: обычного в медный Шалкеровый ящик
+item.ironchest.shulker_box.iron_gold.tooltip= 
+item.ironchest.shulker_box.diamond_obsidian.name=Parannus: алмазного в обсидиановый Шалкеровый ящик
+item.ironchest.shulker_box.iron_gold.tooltip= 
+
+##############
+#    GUIs    #
+##############
+
+itemGroup.ironchest=Rautaineniset arkut
```
Comment (jikuja) [inline src/main/resources/assets/ironchest/lang/fi_FI.lang]: Fix typo

File: src/main/resources/assets/ironchest/lang/fi_FI.lang (hunk 2 above)
Comment (syorito-hatsuki) [reply src/main/resources/assets/ironchest/lang/fi_FI.lang]: Parranus

File: src/main/resources/assets/ironchest/lang/fi_FI.lang (hunk 1 above)
Comment (syorito-hatsuki) [reply src/main/resources/assets/ironchest/lang/fi_FI.lang:12]: I do translate for better listening in finnish and I asking from finnish friend when I do this translates

File: src/main/resources/assets/ironchest/lang/fi_FI.lang (hunk 3 above)
Comment (syorito-hatsuki) [reply src/main/resources/assets/ironchest/lang/fi_FI.lang]: raudasta kultaisen

File: src/main/resources/assets/ironchest/lang/fi_FI.lang (hunk 4 above)
Comment (syorito-hatsuki) [reply src/main/resources/assets/ironchest/lang/fi_FI.lang]: https://fi.wikipedia.org/wiki/Kristalli

Yes. Here I do wrong translate. Correct is Kristalli

File: src/main/resources/assets/ironchest/lang/fi_FI.lang (hunk 5 above)
Comment (syorito-hatsuki) [reply src/main/resources/assets/ironchest/lang/fi_FI.lang]: I from russian, and me simple translate from russian with check english. I do it at night and didnt see this line

File: src/main/resources/assets/ironchest/lang/fi_FI.lang (hunk 6 above)
Comment (syorito-hatsuki) [reply src/main/resources/assets/ironchest/lang/fi_FI.lang]: Ok

File: src/main/resources/assets/ironchest/lang/fi_FI.lang (hunk 1 above)
Comment (jikuja) [reply src/main/resources/assets/ironchest/lang/fi_FI.lang:12]: I admit in this case maintaining one style might be hard.

E.g. Kristalliarkku is better than Kristallinen arkku and Obsidiaaniarkku is better than Obsidiaani arkku or Obsidiaaninen arkku.

File: src/main/resources/assets/ironchest/lang/fi_FI.lang (hunk 5 above)
Comment (jikuja) [reply src/main/resources/assets/ironchest/lang/fi_FI.lang]: 7 lines

File: src/main/resources/assets/ironchest/lang/fi_FI.lang (hunk 1 above)
Comment (syorito-hatsuki) [reply src/main/resources/assets/ironchest/lang/fi_FI.lang:12]: Ok. You from finland and you now better

File: src/main/resources/assets/ironchest/lang/fi_FI.lang (hunk 5 above)
Comment (syorito-hatsuki) [reply src/main/resources/assets/ironchest/lang/fi_FI.lang]: Yesterday I think I translate all. But maybe forget save before close Visual Studio Code

File: src/main/resources/assets/ironchest/lang/fi_FI.lang (hunk 3 above)
Comment (syorito-hatsuki) [reply src/main/resources/assets/ironchest/lang/fi_FI.lang]: Thats bettered?

**Comment by syorito-hatsuki** [comment]
Added fixed translate

File: src/main/resources/assets/ironchest/lang/fi_FI.lang (hunk 6 above)
Comment (syorito-hatsuki) [reply src/main/resources/assets/ironchest/lang/fi_FI.lang]: If you want, check new file. Now I do it so careful

File: src/main/resources/assets/ironchest/lang/fi_FI.lang [hunk 7]
```diff
@@ -0,0 +1,177 @@
+################
+#    Chests    #
+################
+
+tile.ironchest.chest.iron.name=Rauta arkku
```
Comment (Rosentti) [inline src/main/resources/assets/ironchest/lang/fi_FI.lang:5]: These should be
Rautainen
Kultainen
Timanttinen
Kuparinen
Hopea-arkku
Multa-arkku
The rest seems ok. I'm from Finland and am Finnish so those are my tips.

Comment (syorito-hatsuki) [reply src/main/resources/assets/ironchest/lang/fi_FI.lang:5]: Kyl mie kin on suomesta mut ei vaan kina tuu sita kultaista pistetta loydettya, ku on kysynyt kavereilta niin mielet jakautuivat kahtia. Niin paattin jatta nain

**Pull request closed without merging**